Overview.src.html

```diff
@@ -2847,11 +2857,13 @@ <h4 id="should-response-to-request-be-blocked-due-to-nosniff?"><dfn title="shoul
  <!-- XXX xref HTTP cache -->
 
  <li>
-  <p>If <var>credentials flag</var> is set and <var>response</var>'s
-  <span title=concept-response-header-list>header list</span> contains one or more
-  <span title=concept-header>headers</span> <span title=concept-header-name>named</span>
-  `<code title>Set-Cookie</code>`, update the cookies. <span data-anolis-ref>COOKIES</span>
-  <!-- XXX proper hook? -->
+  <p>If <var>credentials flag</var> is set, and the user agent is not configured to block cookies for
+  <var>httpRequest</var> (see <a href="https://tools.ietf.org/html/rfc6265#section-7">Section 7</a> of
+  <span data-anolis-ref>COOKIES</span>), then execute the "set-cookie-string" algorithm defined in
+  <a href="https://tools.ietf.org/html/rfc6265#section-5.2">Section 5.2</a> of
+  <span data-anolis-ref>COOKIES</span> on the <span title=concept-header-value>value</span>
+  of each <var>header</var> <span title=concept-header-name>named</span> `<code title>Set-Cookie</code>`
+  contained in <var>response</var>'s <span title=concept-response-header-list>header list</span>.
```

Comment:
The alternative would be s/contained // and adding ", if any" at the end. I think that's somewhat more consistent with what we do elsewhere.